Uplift/Views/ListCollectionViewCell.swift

```diff
@@ -128,6 +128,25 @@ class ListCollectionViewCell<T, U: ListItemCollectionViewCell<T>>: UICollectionV
     func collectionView(_ collectionView: UICollectionView, didUnhighlightItemAt indexPath: IndexPath) {
         didUnhighlightItemAt(collectionView, indexPath: indexPath)
     }
+    
+    func reloadConfig() {
+        layout.scrollDirection = config.scrollDirection
+        if config.scrollDirection == .vertical {
+            layout.minimumLineSpacing = config.minimumLineSpacing
+        } else {
+            layout.minimumInteritemSpacing = config.minimumInteritemSpacing
+        }
```

Comment:
prefer ternary:
```
layout.minimumLineSpacing = config.scrollDirection == .vertical
  ? config.minimumLineSpacing
  : config.minimumInteritemSpacing
```

Reply:
The `if` and `else` clause assign values to different variables so ternary wouldn't work here.